src/main/java/roomescape/controller/AdminController.java

```diff
@@ -0,0 +1,17 @@
+package roomescape.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class AdminController {
```

Comment:
여기도 `@RequestMapping` 을 사용할 수 있겠네요 😄 

```suggestion
@Controller
@RequestMapping("/admin")
public class AdminController {
```